### DISCUSSION TITLE
Cannot find interface declaration for 'GADInterstitial', superclass of 'RNFBGADInterstitial'

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [invertase/react-native-firebase#4635] [Q&A] [answered]
I am using ReactNative Firebase AdMob. 

`
Cannot find interface declaration for 'GADInterstitial', superclass of 'RNFBGADInterstitial'
`
 This issue arises when I try to build. How can i solve this?

**Comment by mikehardy** [comment]
You opened a discussion, but this is an issue is it not? I'll need the information from the issue template in order to help, so please open an issue

Prior to trying that though, you'll want to make sure you are using fully up to date versions (v10.1.0 here, firebase-ios-sdk 7.1.0, firebase-android-sdk 26.1.0) and you've got a clean build (`npx react-native-clean-project`)

You may also try a clean scripted skeleton project where you can reproduce without interference from project-specific issues: https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh

**Comment by a-c-sreedhar-reddy** [comment, marked as answer]
@mikehardy I have opened an isssue here. https://github.com/invertase/react-native-firebase/issues/4638
will close this discussion
